src/plugins/visualizations/public/vis_types/types_service.ts

```diff
@@ -101,6 +101,21 @@ export class TypesService {
        * returns all registered aliases
        */
       getAliases: visTypeAliasRegistry.get,
+      /**
+       * unregisters a visualization alias by its name
+       * alias is a visualization type without implementation, it just redirects somewhere in kibana
+       * @param {string} visTypeAliasName - visualization alias name
+       */
+      unRegisterAlias: visTypeAliasRegistry.remove,
+      /**
+       * returns all visualizations of specific group
+       * @param {VisGroups} group - group type (aggbased | other | tools)
+       */
+      getByGroup: (group: VisGroups) => {
+        return [...Object.values(this.types)].filter((type) => {
```

Comment:
nit: are you sure we need to copy array using `[...]`. It's not a deep clone 
I think 
`Object.values(this.types).filter((type) => type.group === group)` should works the same

Reply:
You are totally right 🙂